client/components/fixed-navigation-header/README.md

```diff
@@ -1,22 +1,26 @@
-# FixedNavigationHeader (JSX)
+# FixedNavigationHeader (TSX)
```

Comment:
Is this an intentional change? If yes, why TSX?

Reply:
Yes it was, cause it's a typescript component.